test/extended/openstack/cloud-provider-config.go

```diff
@@ -124,3 +188,16 @@ func getConfig(kubeClient kubernetes.Interface, namespace string, cm_name string
 	}
 	return cfg, nil
 }
+
+// return an harmonized string with the value of a property defined inside a section.
+func getPropertyValue(section_name string, property_name string, cfg *ini.File) (string, error) {
```

Comment:
I think I'd add error handling here directly so that we don't need to repeat it when running this function. It's private anyway.

Reply:
It is private but can be reused for other future tests in the file. As I mention before, I prefer to see the exceptions on the test itself, it is better for debugging.